src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/set_interior_vehicle_data_request.cc

```diff
@@ -193,9 +173,26 @@ bool CheckLightDataByCapabilities(
   return true;
 }
 
+bool CheckLightNameByCapabilities(
+    const smart_objects::SmartObject& capabilities_status,
+    const smart_objects::SmartObject& light_data) {
+  LOG4CXX_AUTO_TRACE(logger_);
+  auto it = capabilities_status.asArray()->begin();
+  for (; it != capabilities_status.asArray()->end(); ++it) {
+    smart_objects::SmartObject& so = *it;
+    int64_t current_id = so[message_params::kName].asInt();
+    if (current_id == light_data[message_params::kId].asInt()) {
+      return CheckLightDataByCapabilities(so, light_data);
+    }
+  }
+  LOG4CXX_DEBUG(logger_, "There is no such light name in capabilities");
+  return false;
```

Comment:
@LitvinenkoIra  I think, you can write : 
```
  auto begin = capabilities_status.asArray()->begin();
  auto end = capabilities_status.asArray()->end();
  auto it = std::find(begin, end, name);
  return it == end ? false : CheckLightDataByCapabilities(*it, light_data);
```

Reply:
@BSolonenko Good point  but with this approach we have to sacrifice logging

Reply:
@BSolonenko in this case capabilities_status is array of smart objects like 
```
[
    {
         "densityAvailable":true,
          "name":"FRONT_LEFT_HIGH_BEAM",
          "sRGBColorSpaceAvailable":true
    },
  ....
]
```
and i need to find appropriate name, so i think std::find will not work in this case